dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+                "-inputTests", dtResults + "/orig-order.txt"};
+        ImpactMain.main(args);
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Cleaning Up Files");
+        try {
+            FileUtils.moveDirectory(new File(dtSubjectSource + "/sootTestOutput"), new File(dtResults + "/sootTestOutput-orig-selection"));
+            FileUtils.deleteDirectory(new File(dtSubjectSource + "/sootOutput"));
+        } catch (Exception e){
+            e.printStackTrace();
+        }
+
+        try {
+            FileUtils.moveDirectory(new File(dtSubjectSource + "/selectionOutput"), new File(dtResults + "/selectionOutput"));
+        } catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+    // Setup A Subject For Test Selection
+    private void setupTestParallelization(MavenProject project){
+        TestPluginPlugin.mojo().getLog().info("Test Parallelization: Calculating Runtimes For Tests");
+        args = new String[]{
+                "-classpath", dtLibs + ":" + dtTools + ":" + dtClass + ":" + dtTests,
+                "-inputTests", dtResults + "/orig-order.txt",
+                "-getTime"};
+        PrintStream stdout = System.out;
+        try {
+            PrintStream out = new PrintStream(new FileOutputStream("orig-time.txt"));
+            System.setOut(out);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        ImpactMain.main(args);
+        System.setOut(stdout);
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Cleaning Up Files");
+        try {
+            FileUtils.moveFile(new File(dtSubjectSource + "/orig-time.txt"), new File(dtResults + "/orig-time.txt"));
+        } catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+    // Run Precomputed Dependencies
+    private void runPrecomputedDependencies(MavenProject project){
+        String classpath = dtLibs  + ":" + dtTools + ":" + dtTests + ":" + dtClass;
+
+        runTestPrioritization(project, classpath);
+        runTestParallelization(project, classpath);
+
+        TestPluginPlugin.mojo().getLog().info("Generating dt-lists");
+        args = new String[]{
+                "-prioDirectory", prioResults,
+                "-paraDirectory", paraResults,
+                "-prioOutputDirectory", prioDTLists,
+                "-seleOutputDirectory", seleDTLists,
+                "-paraOutputDirectory", paraDTLists};
+        OutputPrecomputedDependences.main(args);
+    }
+
+    // Run Test Prioritization
+    private void runTestPrioritization(MavenProject project, String classpath){
+        // env-files Directory
+        try {
+            new File(dtResults + "/env-files").createNewFile();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Test Prioritization: Generating Pre-computed Dependencies");
+        for (String k : TESTTYPES) {
```

Comment:
Minor: `TESTTYPES` -> `TEST_TYPES`

Also, should use more descriptive names than `k`, `i`, and `j` (especially given the weird ordering, normally it would at least be `i`, `j`, `k`). I know the scripts did this, but those shouldn't have been like that either.